<launch>

<rosparam command="load" file="$(find control)/config/sign_params.yaml" />
<arg name="show" default="false"/>
<arg name="print" default="false"/>
<arg name="print_duration" default="false"/>
<arg name="hasDepthImage" default="false"/>
<node pkg="control" name="signFastest" output="screen" type="signFastest" required="true">
    <param name="showFlag" value="$(arg show)"/>
    <param name="printFlag" value="$(arg print)"/>
    <param name="printDuration" value="$(arg print_duration)"/>
    <param name="hasDepthImage" value="$(arg hasDepthImage)"/>
</node>

</launch>